autorest/date/unixtime_test.go

```diff
@@ -1,3 +1,5 @@
+// +build go1.7
```

Comment:
> Exclude the unixtime tests from 1.6 as they rely on functionality that
isn't available in 1.6.

Whoops, I should read _gooder_. However, I am still curious about what that means for the packages that rely on UnixTime.